src/pages/Payment.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import "./Payment.css"; // 引入支付页面的 CSS 文件
+import { useParams } from "react-router-dom";
+
+const Payment = () => {
+  const { amount } = useParams();
+  const [paymentMessage, setPaymentMessage] = useState(""); // 弹窗消息的状态
+
+  const handlePayment = async (e) => {
+    e.preventDefault();
+    console.log("Payment amount:", amount);
+    try {
+      const amountFloat = parseFloat(amount);
+      // 发送支付请求
+      const response = await fetch(
+        "http://localhost:8000/api/handle_payment/",
+        {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            amount: amountFloat,
+
+            // 其他必要的支付信息
+          }),
+        }
+      );
+      if (!response.ok) {
+        throw new Error("Payment failed. Please try again.");
+      }
+      const data = await response.json();
+      // 显示支付成功消息
+      setPaymentMessage(data.message);
+    } catch (error) {
+      // 显示支付失败消息
+      setPaymentMessage(error.message);
```

Comment:
since the handle_payment API is not working at the moment, i recommend displaying all payment attempts to be successful. we can go back to fix the API later